routers/routes/routes.go

```diff
@@ -340,6 +341,8 @@ func RegisterRoutes(m *macaron.Macaron) {
 		})
 	}, reqSignOut)
 
+	m.Any("/user/events", reqSignIn, events.Events)
```

Comment:
What verb(s) are actually used here?

Reply:
Good question - That is why I put Any. Chrome appears to use GET.

Reply:
Well... since we will be ignoring the verb I don't think it poses any kind of security problem. If some obscure version of IE for industrial X-Box washing machines tries to use `PATCH`, it would be its problem. 😆